.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] Kindly create `test.md file` and add it to` .gitignore` as well according to the project requirement. 👍

Reply:
Fixed.